### PR TITLE
About page michael

### PR DESCRIPTION
**Pull request by kaka5277**
## Rationale
- This is for testing the git workflow

## Advice for Reviewers & Testing Notes 
- Explain how your change should be reviewed, and where reviewers should direct their attention.
- Explain (where relevant) how your change can be tested.

## Screenshots:
- Add a screenshot or two of your changes, where applicable.

## Task Name
- Issue Title

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -408,7 +409,7 @@ <h4 class="title">
 =======
                             Over 25 years working in tax services developing x  and x for
 >>>>>>> origin/about-page-michael
```
Comment (davidtaing) [inline about.html]: https://github.com/codesydney/aicta/blob/804b53d1e289ae5f5cf4f182459f5d5b29fe1e19/about.html#L407-L411

Opps, looks like there was a merge conflict marker. Sorry, I should have picked this up last time.

Can you please do me a favour and fix up the lines above?

**Review by davidtaing — CHANGES_REQUESTED** [review]
Looks good to me, @kaka5277! Thanks for putting this through.

Can please help me out and fix up the merge conflict below. Once that's done, please merge in your changes.

**Comment by kaka5277** [comment]
hi dave，
&nbsp; i‘m trying how to resolve merge conflict.i'll let you know when i make it or not.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "codesydney/aicta"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月18日(星期六) 下午2:14
***@***.***&gt;;
抄送:&nbsp;"Jun ***@***.******@***.***&gt;;
主题:&nbsp;Re: [codesydney/aicta] About page michael (PR #10)





 
@davidtaing requested changes on this pull request.
 
Looks good to me, @kaka5277! Thanks for putting this through.
 
Can please help me out and fix up the merge conflict below. Once that's done, please merge in your changes.

 
In about.html:
 &gt; @@ -408,7 +409,7 @@ <h4 class="title"&gt;  =======                              Over 25 years working in tax services developing x  and x for  &gt;&gt;&gt;&gt;&gt;&gt;&gt; origin/about-page-michael  
https://github.com/codesydney/aicta/blob/804b53d1e289ae5f5cf4f182459f5d5b29fe1e19/about.html#L407-L411
 
Opps, looks like there was a merge conflict marker. Sorry, I should have picked this up last time.
 
Can you please do me a favour and fix up the lines above?

File: about.html (hunk 1 above)
Comment (kaka5277) [reply about.html]: hi dave,should be resolved the conflict.just check it.

**Review by davidtaing — APPROVED** [review]

**Comment by davidtaing** [comment]
I've merged in the changes for this one. Sorry about the delay.

**Pull request merged**